js/views/helper.js

```diff
@@ -102,6 +102,13 @@ define(function(require) {
 		return t('mail', text);
 	});
 
+	Handlebars.registerHelper('printTotalOrUnseenCount', function(folder, unseen, total) {
+		if(folder == 'drafts') {
+			return total;
```

Comment:
jscs reports here: "requireSpaceAfterKeywords: One space required after "if" keyword at". You should install jscs and test your style.